tools/niminst/install.tmpl

```diff
@@ -3,7 +3,16 @@
 #  result = "#! /bin/sh\n# Generated by niminst\n"
 #  var proj = c.name.toLower
 
-set -e
+BASE_DIR=$(pwd)
+NIM_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+
+function go_back {
```

Comment:
Ok, I'll check it for POSIX compatibility and improve if possible.

С уважением,
Денис Колодин
On 25 Jun 2016 10:49 a.m., "Andreas Rumpf" <notifications@github.com> wrote:

In tools/niminst/install.tmpl
<https://github.com/nim-lang/Nim/pull/4410#discussion_r68485823>:

> @@ -3,7 +3,16 @@
>  #  result = "#! /bin/sh\n# Generated by niminst\n"
>  #  var proj = c.name.toLower
>
> -set -e
> +BASE_DIR=$(pwd)
> +NIM_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
> +
> +function go_back {

I don't think 'function' is in the Posix standard. Note that this script is
Posix, not Bash.

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub
<https://github.com/nim-lang/Nim/pull/4410/files/a754460f832809f19fce4a6644b7b34c5cd163ec#r68485823>,
or mute the thread
<https://github.com/notifications/unsubscribe/AAZkaJRusXctNnD6JPUyW4j2qRbq4wIbks5qPN1xgaJpZM4I-RmD>
.